lib/whitehall_importer.rb

```diff
@@ -19,4 +19,9 @@ def self.import(whitehall_document)
 
     record
   end
+
+  def self.sync(document)
+    ResyncService.call(document)
+    ClearLinksetLinks.call(document.content_id)
+  end
```

Comment:
Looks good 👍 I think this will have to grow in sophistication with https://trello.com/c/GbgBv1Nz/1233-track-and-implement-more-statuses-in-whitehall-import-aborted-import-failed-syncing-sync-failed-complete where it'll need to take the import object rather than a document.